tests/ports/unix/mpy_coverage_rv32imc.py

```diff
@@ -1,3 +1,49 @@
 import mpy_coverage
 
 mpy_coverage.run_one_arch("rv32imc", "rv32")
+
+print("Expected to compile")
+mpy_coverage.run_extra_cases(
+    "rv32imc",
+    [
+        "@micropython.asm_rv32\ndef f(): ebreak()",
+    ],
+)
+
+# Test "argument out of range" specifically for label targets
+nl = "\n"
+beqz_test = f"""
+@micropython.asm_rv32
+def f():
+ c_beqz(a0, l)
+{nl.join(" ecall()" for _ in range(64))}
+ label(l)
+ """
+
+print("Expected to fail")
+mpy_coverage.run_extra_cases(
+    "rv32imc",
+    [
+        "@micropython.asm_rv32\ndef f(a0, a1, a2, a3, a4): ac_li(a0, 1)",
```

Comment:
There's already `tests/inlineasm/rv32/asmsanity.py`.  Tests that exercise out of bounds values or check against zero or whatnot should go there instead?

Probably some of these are already replicated there.

Reply:
I agree moving it to inlineasm/ probably makes sense.

I believe these were all "new coverage" because mpy_coverage_rv32imc is supposed to be running all the existing tests in inlineasm/rv32 and I added each of these cases where this PR saw missing coverage in the C code.